svelte/components/Nft/Nft.svelte

```diff
@@ -50,6 +48,9 @@
 
     // STATE VIEW : sync get Nft
     nft = nftStore.getOneStore(chainId, address, tokenID);
+    if (!$nft?.collection?.supports) {
```

Comment:
first ? not necassary => `$nft.collection?.supports`

Reply:
Without it it causes an error when using hash of the nft in the url